lib/components/molecules/Pagination/Pagination.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+/**
+ *
+ * Pagination
+ *
+ */
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+
+import styles from './Pagination.style';
+import type { Node } from './types';
+import { noop } from '@babel/types';
+
+type Props = {
+  size: number,
+  render: Function,
+  items: Array,
+  className: Array | string,
+};
+const buttons = (n, component) => Array.from('x'.repeat(n), (_, s) => component(s + 1));
```

Comment:
Array.from is not yet supported by IE 11, whats the fallback for it?

Reply:
Can use a core.js polyfill